client/components/badge/style.scss

```diff
@@ -15,3 +15,8 @@
 	color: var( --color-white );
 	background-color: var( --color-success );
 }
+
+.badge--info {
```

Comment:
@shaunandrews Is `--info` the right terminology for this variation? (I looked briefly but didn't see a Material standard for this.)

Also, please verify the color choices.

Reply:
I'm cool with `--info`. Color, font-size, and spacing looks good to my eyes.

Reply:
Thanks!